woocommerce/woocommerce-ios
# Roll back UL&S

Closes #3394 
For context, see For context: p91TBi-44z-p2

This PR rolls back the Login flow to the previous implementation, pre Unified Login. The rollback is implemented in one commit, so, in theory, activating UL&S again is a matter of reverting that commit. 

Sorry for the PR size. In between xibs, tests, and code removals the diff is quite large. 

Just in case, for extra piece of mind, I have pushed a branch containing the Unified Login flow: `backup/unified-login` 

## Changes
- Point WPAuthenticator to 1.28.0
- Pin WordPressKit to 4.23.0. Otherwise WPAuthenticator pulls a beta version of WordPressKit that WPAuthenticator does not actually support. This does not seem like the right moment to spend time fixing the dependency between WPAuthenticator and WPKit 
- Revert LoginPrologueViewController.swift and LoginPrologueViewController.xib
- Remove code that will not build when reverting the dependency with WPAuthenticator.
- Revert changes to StorePickerViewController
- Revert changes to UI tests.
- Updated release notes.

## How to test
* Checkout the branch, run `bundle exec pod install`
* Log out if necessary. Attempt to log in. Notice the login flow is the same as 5.6 (the version available in the App Store at the moment)


Update release notes:
